andela/hc-leo
# [Feature #157694807] Users priority list

**What does this PR do?**
Adds the ability for a team owner to assign priorities to team members which determines the order in which they will receive notifications. The priorities are High and Low.

<img width="610" alt="screen shot 2018-06-20 at 18 13 55" src="https://user-images.githubusercontent.com/25581395/41667551-ecf0d722-74b5-11e8-90d3-bc63356d4b63.png">

**Description of task to be accomplished**
User should be able to set up a list of users(priority list) to be notified one after the other when a job fails until the alert is resolved

**How should this be manually tested**
- Log in to healthchecks. Make sure you have team access enabled (This can be enabled by setting the field `team_access_allowed` for your account to True in the profile table)
- Navigate to account settings.
- In the team access section, invite two or more team members.
- Assign each member a priority.
- The team members should receive alerts according to their priority.

**Relevant stories**
[#157694807](https://www.pivotaltracker.com/story/show/157694807)
